cats/cats-core/src/main/java/com/netflix/spinnaker/cats/mem/InMemoryCache.java

```diff
@@ -332,7 +332,7 @@ private void handleDelim(String s) {
                             toInit();
                             break;
                         default:
-                            throw new IllegalStateException("Unhandled delimiter in init state: " + s);
+                            throw new IllegalStateException(String.format("Unhandled delimiter in init state: %s", s));
```

Comment:
Odd - must be something specific to your jvm version or something (@ttomsu had disabled it at one point mentioning this line). It works for me and works on Travis-CI

Feel free to ignore and I will see if I can figure out the cause and a way to conditionally disable it instead of blanket disabling